src/models/Puck.js

```diff
@@ -3,22 +3,24 @@ import { ELECTRIC_CHARGE_TYPE } from '../const/charge.const';
 
 export default class Puck extends ElectricCharge {
   constructor(x, y) {
-    super(x, y, ELECTRIC_CHARGE_TYPE.NEGATIVE);
+    super(x, y, ELECTRIC_CHARGE_TYPE.POSITIVE);
     this.reset();
   }
 
   reset() {
     this.velocity = { x: 0, y: 0 };
     this.acceleration = { x: 0, y: 0 };
+    this.x = 100;
+    this.y = 300;
```

Comment:
I don't think resetting position should be included in the reset function here as the x and y parameters are passed in constructor thus they come from an external source and are not a part of deterministic initial state of this object. Additionality, call to the super constructor sets x and y values which then get overridden by the values inside of reset(). I suggest resetting the Puck's position in Game.js, keeping in mind to use constants. Also you should use `BoudingBox.move(x, y)` instead of directly setting x and y values, because moving the BoudingBox requires you to also update `right` and `bottom` fields.

```suggestion
    // In Game.js
    reset() {
      // ...
      this.puck.move(PUCK_INITIAL_X, PUCK_INITIAL_Y);
    }
```